### PR TITLE
Javadoc

### PR DESCRIPTION
**Pull request by holkwen**

### CODE REVIEW

File: src/main/java/net/greatstart/controllers/InvestmentController.java [hunk 1]
```diff
@@ -41,14 +47,23 @@ public InvestmentController(InvestmentService investmentService,
...
+    @GetMapping("/{id}")
     public ResponseEntity<DtoInvestment> getInvestmentById(@PathVariable long id) {
         if (investmentService.getDtoInvestmentById(id) != null) {
             return new ResponseEntity<>(HttpStatus.OK);
         }
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
 
+    /*@GetMapping("/my")
+    public ResponseEntity<DtoInvestment> getInvestmentById(Principal principal) {
```
Comment (Sheptytskyid) [inline src/main/java/net/greatstart/controllers/InvestmentController.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/main/java/net/greatstart/controllers/InvestmentController.java [hunk 2]
```diff
@@ -41,14 +47,15 @@ public InvestmentController(InvestmentService investmentService,
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
 
-    @GetMapping("{id}")
+    @GetMapping("/{id}")
```
Comment (zozich) [inline src/main/java/net/greatstart/controllers/InvestmentController.java]: Why?

File: src/main/java/net/greatstart/controllers/InvestmentController.java [hunk 3]
```diff
@@ -33,6 +38,7 @@ public InvestmentController(InvestmentService investmentService,
         this.investmentValidationService = investmentValidationService;
     }
 
+    @Transactional
```
Comment (zozich) [inline src/main/java/net/greatstart/controllers/InvestmentController.java]: Why not on service?

**Review by zozich — APPROVED** [review]
...

**Pull request merged**